src/pages/home/home.scss

```diff
@@ -0,0 +1,73 @@
+.homeContainer {
+  height: 100%;
+
+  & .mainHeaderBackground {
+    background: url('../../assets/main-header-background.png') no-repeat;
+    height: 100%;
+    width: 100%;
+    background-size: cover;
+    position: absolute;
+    z-index: -1;
```

Comment:
Please follow this tutorial (https://jonsuh.com/blog/organizing-z-index-with-sass/) to deal with the z-index